packages/frontend/src/graphql/server.js

```diff
@@ -0,0 +1,60 @@
+import gql from 'graphql-tag'
+
```

Comment:
Most of the graphql query definition changes can be summarized as follows
1. Using a .js file for defining queries is better, because you can 1) have multiple queries in the same file 2) compose queries from various fragments, increasing DRYness
2. Defining standard queries for optimized cache usage. Instead of executing 3 different user/serverInfo queries causing unnecessary requests, have all components rely on the same query so that cache can be re-used. (I was able to reduce GQL request amount on Streams page from 9 to 3)